domain/src/main/java/com/teamagam/gimelgimel/domain/timeplay/DatePickerOpener.java

```diff
@@ -0,0 +1,5 @@
+package com.teamagam.gimelgimel.domain.timeplay;
+
+public interface DatePickerOpener {
+  void showPicker();
```

Comment:
show is enough.
Since it's a DatePickerOpener it is clear that it shows a picker.

Reply:
You can consider renaming Opener to DialogShower to be a little more consistent with android terminology

Reply:
done